### PR TITLE
Update list permissions level for run-only flow users

### PR DESCRIPTION
**Pull request by Steven-Jia**
Also updating SharePoint list to say Microsoft list

## Category

- [x] Content fix
- [ ] New article

## What's in this Pull Request?

Updating guidance on list permissions needed for run-only users to run flows on the list. Also updating "SharePoint list" to "Microsoft list" to reflect the new product branding.

### CODE REVIEW

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[07c0e52](https://github.com/Steven-Jia/sp-dev-docs-edits/commits/07c0e527414f51cbd851865bb09feeb0a787d583)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/business-apps/power-automate/guidance/manage-list-flows.md](https://github.com/Steven-Jia/sp-dev-docs-edits/blob/patch-1/docs/business-apps/power-automate/guidance/manage-list-flows.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-automate/guidance/manage-list-flows?branch=pr-en-us-7037) | [Details](#user-content-4f5250b57f28dfa7e080eea4e94d8f4e-f4a2d10e)

<a id="4f5250b57f28dfa7e080eea4e94d8f4e-f4a2d10e"></a>
### [docs/business-apps/power-automate/guidance/manage-list-flows.md](https://github.com/Steven-Jia/sp-dev-docs-edits/blob/patch-1/docs/business-apps/power-automate/guidance/manage-list-flows.md)
 - **Line 2, Column 1**: **[Suggestion-[description-missing](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````
 - **Line 42, Column 192**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/sharepoint/understanding-permission-levels#list-permissions-and-permission-levels' will be broken in isolated environments. Replace with a relative link.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C5%5C25%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202105251955456930-7037%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=TqruQDScFCWbCV7QFOOYRrcGRbIYRG1mnS7R%2Fv4e0p4%3D&st=2021-05-25T19%3A51%3A47Z&se=2021-06-25T19%3A56%3A47Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[90058cb](https://github.com/Steven-Jia/sp-dev-docs-edits/commits/90058cb7234d7b0ee10fc237b50236117425baef)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/business-apps/power-automate/guidance/manage-list-flows.md](https://github.com/Steven-Jia/sp-dev-docs-edits/blob/patch-1/docs/business-apps/power-automate/guidance/manage-list-flows.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-automate/guidance/manage-list-flows?branch=pr-en-us-7037) | [Details](#user-content-4f5250b57f28dfa7e080eea4e94d8f4e-d577928c)

<a id="4f5250b57f28dfa7e080eea4e94d8f4e-d577928c"></a>
### [docs/business-apps/power-automate/guidance/manage-list-flows.md](https://github.com/Steven-Jia/sp-dev-docs-edits/blob/patch-1/docs/business-apps/power-automate/guidance/manage-list-flows.md)
 - **Line 2, Column 1**: **[Suggestion-[description-missing](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````
 - **Line 42, Column 188**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/sharepoint/understanding-permission-levels#list-permissions-and-permission-levels' will be broken in isolated environments. Replace with a relative link.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C5%5C25%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202105251958298584-7037%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=FAU%2BQgYIgyMxOZUWpSOtMb3g7C%2BcVXkElncrNL4fOPI%3D&st=2021-05-25T19%3A54%3A11Z&se=2021-06-25T19%3A59%3A11Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/business-apps/power-automate/guidance/manage-list-flows.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
-title: Manage owners and users in your SharePoint list flows with Power Automate
+title: Manage owners and users in your Microsoft list flows with Power Automate
 ms.date: 05/19/2020
```
Comment (andrewconnell) [inline docs/business-apps/power-automate/guidance/manage-list-flows.md:3]: Can you please update the doc date to reflect when things were changed?

File: docs/business-apps/power-automate/guidance/manage-list-flows.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 search.app: 
   - Flow
 search.appverid: met150
 ---
```
Comment (andrewconnell) [inline docs/business-apps/power-automate/guidance/manage-list-flows.md:7]: Please add a `description` metadata property as required for SEO by MSDocs

File: docs/business-apps/power-automate/guidance/manage-list-flows.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ For triggers that support run-only users, below the **Owners** card, a **Manage
 
 ![Manage run-only users card dropdown](../../../images/manage-run-only-users-card.png)
 
-Here, you can manage the run-only users or assign the users of the SharePoint list as run-only users. This means users who have read permissions to that list can run flows.
+Here, you can manage the run-only users or assign the users of the Microsoft list as run-only users. In order to run flows, users must have Edit Permissions on the list. Learn more about [list permissions and permission levels](https://docs.microsoft.com/sharepoint/understanding-permission-levels#list-permissions-and-permission-levels).
```
Comment (andrewconnell) [inline docs/business-apps/power-automate/guidance/manage-list-flows.md:42]: Please convert the absolute URL to a relative link, per MSDocs red’s and listed in the build check report in the PR comments.

**Review by andrewconnell — CHANGES_REQUESTED** [review]
There are a few things that need to get addressed before we can merge this. Can you take a look at the review? Thanks!

File: docs/business-apps/power-automate/guidance/manage-list-flows.md (hunk 3 above)
Comment (Steven-Jia) [reply docs/business-apps/power-automate/guidance/manage-list-flows.md:42]: How do you use a relative URL to refer to an article that is not in the dev docs, but another part of docts. microsoft.com?

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[1989f50](https://github.com/Steven-Jia/sp-dev-docs-edits/commits/1989f505dd5c272b10f200ac648ad7b8d544bbda)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/business-apps/power-automate/guidance/manage-list-flows.md](https://github.com/Steven-Jia/sp-dev-docs-edits/blob/patch-1/docs/business-apps/power-automate/guidance/manage-list-flows.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-automate/guidance/manage-list-flows?branch=pr-en-us-7037) | [Details](#user-content-4f5250b57f28dfa7e080eea4e94d8f4e-cc02133e)

<a id="4f5250b57f28dfa7e080eea4e94d8f4e-cc02133e"></a>
### [docs/business-apps/power-automate/guidance/manage-list-flows.md](https://github.com/Steven-Jia/sp-dev-docs-edits/blob/patch-1/docs/business-apps/power-automate/guidance/manage-list-flows.md)
 - **Line 43, Column 188**: **[Warning-[file-not-found](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../../../../sharepoint/understanding-permission-levels'.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C6%5C2%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202106022053049074-7037%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=32Ie9iC%2FHjzg3xXcbkDDJjJJHtvRLhojoEGpdbQd1kY%3D&st=2021-06-02T20%3A49%3A21Z&se=2021-07-03T20%3A54%3A21Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[eeedeb7](https://github.com/Steven-Jia/sp-dev-docs-edits/commits/eeedeb7f000040b077f379a7ccdff0075dbaae6f)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/business-apps/power-automate/guidance/manage-list-flows.md](https://github.com/Steven-Jia/sp-dev-docs-edits/blob/patch-1/docs/business-apps/power-automate/guidance/manage-list-flows.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-automate/guidance/manage-list-flows?branch=pr-en-us-7037) | [Details](#user-content-4f5250b57f28dfa7e080eea4e94d8f4e-10dcd84c)

<a id="4f5250b57f28dfa7e080eea4e94d8f4e-10dcd84c"></a>
### [docs/business-apps/power-automate/guidance/manage-list-flows.md](https://github.com/Steven-Jia/sp-dev-docs-edits/blob/patch-1/docs/business-apps/power-automate/guidance/manage-list-flows.md)
 - **Line 43, Column 188**: **[Warning-[file-not-found](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../../../../../sharepoint/understanding-permission-levels'.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C6%5C2%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202106022054455625-7037%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=bRO4JVCvnYtqeVixyCJBZZF%2FbNH8wq452oX5Ed8LEOQ%3D&st=2021-06-02T20%3A50%3A48Z&se=2021-07-03T20%3A55%3A48Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[0a69790](https://github.com/Steven-Jia/sp-dev-docs-edits/commits/0a69790dd24cda9a19f3c63bc1d7ff4142803c01)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/business-apps/power-automate/guidance/manage-list-flows.md](https://github.com/Steven-Jia/sp-dev-docs-edits/blob/patch-1/docs/business-apps/power-automate/guidance/manage-list-flows.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-automate/guidance/manage-list-flows?branch=pr-en-us-7037) | [Details](#user-content-4f5250b57f28dfa7e080eea4e94d8f4e-9d260586)

<a id="4f5250b57f28dfa7e080eea4e94d8f4e-9d260586"></a>
### [docs/business-apps/power-automate/guidance/manage-list-flows.md](https://github.com/Steven-Jia/sp-dev-docs-edits/blob/patch-1/docs/business-apps/power-automate/guidance/manage-list-flows.md)
 - **Line 43, Column 188**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/sharepoint/understanding-permission-levels#list-permissions-and-permission-levels' will be broken in isolated environments. Replace with a relative link.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C6%5C2%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202106022103061629-7037%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=v3nvDfE8UWqtCE%2BGIGng1pZiILGy2axrw8FExt7YQ4M%3D&st=2021-06-02T20%3A59%3A16Z&se=2021-07-03T21%3A04%3A16Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/business-apps/power-automate/guidance/manage-list-flows.md (hunk 3 above)
Comment (Steven-Jia) [reply docs/business-apps/power-automate/guidance/manage-list-flows.md:43]: @andrewconnell I tried using a relative URL but it wasn't working. If there is a way to relatively refer to a doc outside of dev docs, please let me know.

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[a61bb29](https://github.com/Steven-Jia/sp-dev-docs-edits/commits/a61bb29729240e206a5cac9efb57f63ec9f0970e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/business-apps/power-automate/guidance/manage-list-flows.md](https://github.com/Steven-Jia/sp-dev-docs-edits/blob/patch-1/docs/business-apps/power-automate/guidance/manage-list-flows.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-automate/guidance/manage-list-flows?branch=pr-en-us-7037) | [Details](#user-content-4f5250b57f28dfa7e080eea4e94d8f4e-beb15aa4)

<a id="4f5250b57f28dfa7e080eea4e94d8f4e-beb15aa4"></a>
### [docs/business-apps/power-automate/guidance/manage-list-flows.md](https://github.com/Steven-Jia/sp-dev-docs-edits/blob/patch-1/docs/business-apps/power-automate/guidance/manage-list-flows.md)
 - **Line 43, Column 188**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/sharepoint/understanding-permission-levels#list-permissions-and-permission-levels' will be broken in isolated environments. Replace with a relative link.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C6%5C2%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202106022103155113-7037%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=5%2Btb6mfM7K1I8bXYLlh8y3Ac%2F46q33o8ck2cr90DwNc%3D&st=2021-06-02T21%3A00%3A16Z&se=2021-07-03T21%3A05%3A16Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[7ae17d6](https://github.com/Steven-Jia/sp-dev-docs-edits/commits/7ae17d627b44c6fe711e282bfe46f391a7c35478)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/business-apps/power-automate/guidance/manage-list-flows.md](https://github.com/Steven-Jia/sp-dev-docs-edits/blob/patch-1/docs/business-apps/power-automate/guidance/manage-list-flows.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/business-apps/power-automate/guidance/manage-list-flows?branch=pr-en-us-7037) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C6%5C14%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202106141025235357-7037%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=vkh1QJ2Nf3hCIbElyObFpd6gdTuztk8VcCrVFgyzkMs%3D&st=2021-06-14T10%3A21%3A34Z&se=2021-07-15T10%3A26%3A34Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/business-apps/power-automate/guidance/manage-list-flows.md (hunk 3 above)
Comment (andrewconnell) [reply docs/business-apps/power-automate/guidance/manage-list-flows.md]: Just remove the FQDL like you see I've done. When in the same repo, you want to use the relative link to the file to get the automatic link checking by msdocs' automation checks, but in this case, just remove the protocol and domain.

**Pull request merged**